src/GameComponent.js

```diff
@@ -1,6 +1,6 @@
-import React from 'react';
-import firebase from 'firebase';
-import gameData from './gameData.js';
+import React from "react";
```

Comment:
Please revert changes to this base class. Looks like an auto-formatter, but you shouldn't need to touch this file in this pull request.